### ISSUE TITLE
Creating channels and it's future.

### ISSUE THREAD

**Issue by smartclash** [csr/discord-coffee-buddy#8]
_This issue is more of a discussion rather a fix to code._

Let's take this. There are 100 people in server. It is week one. The both matches 100 people where each match will have two people. Basically this means there will be **50** text channels and **50** voice channels created _just_ for the week one pairing session.

Totaling to a whooping 100 channels (both text and voice) for every pairing session. ***Phew!***

This discussion is basically geared towards tackling this problem. Though creating channels has no costs and is completely free, it will be pretty hard for people to keep track of. Especially for the server admins to moderate and manage the channels.

**Comment by csr** [comment]
A simple solution could be simply to send a message to the user with their match and that's it.

**Comment by tayoogunbiyi** [comment]
Perhaps we can delete channels?

**Comment by smartclash** [comment]
Yeah, we could be doing that. We could purge every channel created each week @tayoogunbiyi.

**Comment by tayoogunbiyi** [comment]
I also don’t think 100 people would sign up but it’s good to think about it
